README.md

```diff
@@ -70,13 +70,14 @@ alarm configured for it.
 2. Unless doing an AIO build, prepare the openstack-ansible configuration.
   1. recursively copy the openstack-ansible-deployment configuration files:
      `cp -R openstack-ansible/etc/openstack_deploy /etc/openstack_deploy`
-  2. merge /etc/openstack_deploy/user_variables.yml with rpcd/etc/openstack_deploy/user_variables.yml:
+  2. move OSA variables to the correct locations.
 
      ```
-     scripts/update-yaml.py /etc/openstack_deploy/user_variables.yml rpcd/etc/openstack_deploy/user_variables.yml
+     rm /etc/openstack_deploy/user_variables.yml  # unused, nothing set
+     mv /etc/openstack_deploy/user_secrets.yml /etc/openstack_deploy/user_osa_secrets_defaults.yml
      ```
   3. copy the RPC configuration files:
-     1. `cp rpcd/etc/openstack_deploy/user_extras_*.yml /etc/openstack_deploy`
+     1. `cp rpcd/etc/openstack_deploy/user_*_defaults.yml /etc/openstack_deploy`
```

Comment:
The reason that wasn't included is because it is laid down by the deploy.sh script.  Should I remove it from there and add a manual instruction here?